peepholelib/classifier/tgmm.py

```diff
@@ -60,4 +60,17 @@ def classifier_probabilities(self, **kwargs):
         data, _ = self.parser(data = batch, **self.parser_kwargs)
         probs = torch.tensor(self._classifier.predict_proba(data), dtype=data.dtype)
         return probs  
+    
+    def save(self, **kwargs):
+        self.file_path = self.path/('GMM.' + f'cv_dim={self._classifier.model_.config.num_features}' + '.' + f'num_cluster={self.nl_class}')
```

Comment:
use `self.nl_model` instead of `self._classifier.model_.config.num_features`.
Do not save `file_path` into `self.`